nx migrate not working

### Please provide the environment you discovered this bug in.

windows nx 19.1.0

### Which area/package is the issue in?

create-analog

### Description

Migration fails. 

nx migrate @analogjs/platform@latest fails because @analogjs/trpc is not found in a aligned version. 

The latest version of @analogjs/trpc is 0.2.45 or 1.0.0.beta-5


### Please provide the exception or error you saw

```true
Unable to fetch migrations for @analogjs/trpc@1.4.0: Command failed: npm install @analogjs/trpc@1.4.0
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @analogjs/trpc@1.4.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
```


### Other information

_No response_

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [X] No

--- a/packages/astro-angular/package.json
+++ b/packages/astro-angular/package.json
@@ -50,7 +50,6 @@
       "@analogjs/platform",
       "@analogjs/content",
       "@analogjs/router",
-      "@analogjs/trpc",
       "@analogjs/vite-plugin-angular",
       "@analogjs/vite-plugin-nitro",
       "@analogjs/vitest-angular"

--- a/packages/content/package.json
+++ b/packages/content/package.json
@@ -56,7 +56,6 @@
       "@analogjs/platform",
       "@analogjs/content",
       "@analogjs/router",
-      "@analogjs/trpc",
       "@analogjs/vite-plugin-angular",
       "@analogjs/vite-plugin-nitro",
       "@analogjs/vitest-angular"

--- a/packages/platform/package.json
+++ b/packages/platform/package.json
@@ -42,7 +42,6 @@
       "@analogjs/platform",
       "@analogjs/content",
       "@analogjs/router",
-      "@analogjs/trpc",
       "@analogjs/vite-plugin-angular",
       "@analogjs/vite-plugin-nitro",
       "@analogjs/vitest-angular"

--- a/packages/router/package.json
+++ b/packages/router/package.json
@@ -34,7 +34,6 @@
       "@analogjs/platform",
       "@analogjs/content",
       "@analogjs/router",
-      "@analogjs/trpc",
       "@analogjs/vite-plugin-angular",
       "@analogjs/vite-plugin-nitro",
       "@analogjs/vitest-angular"

--- a/packages/trpc/package.json
+++ b/packages/trpc/package.json
@@ -33,7 +33,6 @@
       "@analogjs/platform",
       "@analogjs/content",
       "@analogjs/router",
-      "@analogjs/trpc",
       "@analogjs/vite-plugin-angular",
       "@analogjs/vite-plugin-nitro",
       "@analogjs/vitest-angular"

--- a/packages/vite-plugin-angular/package.json
+++ b/packages/vite-plugin-angular/package.json
@@ -31,7 +31,6 @@
       "@analogjs/platform",
       "@analogjs/content",
       "@analogjs/router",
-      "@analogjs/trpc",
       "@analogjs/vite-plugin-angular",
       "@analogjs/vite-plugin-nitro",
       "@analogjs/vitest-angular"

--- a/packages/vite-plugin-nitro/package.json
+++ b/packages/vite-plugin-nitro/package.json
@@ -32,7 +32,6 @@
       "@analogjs/platform",
       "@analogjs/content",
       "@analogjs/router",
-      "@analogjs/trpc",
       "@analogjs/vite-plugin-angular",
       "@analogjs/vite-plugin-nitro",
       "@analogjs/vitest-angular"

--- a/packages/vitest-angular/package.json
+++ b/packages/vitest-angular/package.json
@@ -37,7 +37,6 @@
       "@analogjs/platform",
       "@analogjs/content",
       "@analogjs/router",
-      "@analogjs/trpc",
       "@analogjs/vite-plugin-angular",
       "@analogjs/vite-plugin-nitro",
       "@analogjs/vitest-angular"

